rtgui/options.cc

```diff
@@ -2229,10 +2229,13 @@ int Options::saveToFile (Glib::ustring fname)
     FILE *f = g_fopen (fname.c_str (), "wt");
 
     if (f == nullptr) {
-        if (options.rtSettings.verbose) {
-            printf ("Options::saveToFile / Error: unable to open file \"%s\" with write access!\n", fname.c_str());
-        }
-
+        std::cout << "Warning! Unable to save your preferences to: " << fname << std::endl;
+#ifndef RAWTHERAPEE_CLI
+        Glib::ustring msg_ = Glib::ustring::compose(M("MAIN_MSG_WRITEFAILED"), fname.c_str());
```

Comment:
I tried doing as you suggested, but it turns out that getting the failed file path then is not possible, at least not without a bit of coding overkill which is beyond my skill. As such, I will close the issue.